packages/augur-ui/src/modules/reporting/common.styles.less

```diff
@@ -246,6 +246,11 @@ div.ReportingSubheaders {
   @media @breakpoint-mobile {
     width: unset;
   }
+  
+  @media @breakpoint-mobile-new {
```

Comment:
already mentioned, don't like the `@breakpoint-mobile-new` naming isn't descriptive enough